2996_reflection/reflection.md

```diff
@@ -11,3 +11,91 @@ toc: true
 ---
 
 # Introduction
+
+This is a proposal for a reduced initial set of features to support static reflection in C++.
+Specifically, we are mostly proposing a subset of features suggested in P1240R2:
+
+  - the representation of program elements via constant-expressions producing
+     _reflection values_ — _reflections_ for short — of an opaque type `std::meta::info`,
+  - a _reflection operator_ (prefix `^`) that produces a reflection value for its operand construct,
+  - a number of `consteval` _metafunctions_ to work with reflections (including deriving other reflections), and
+  - constructs called _splicers_ to produce grammatical elements from reflections (e.g., `[:` _refl_ `:]`).
```

Comment:
Can use `$` inside of code blocks to italicize, which means you don't have to leave the code block so this renders nicer:
```suggestion
  - constructs called _splicers_ to produce grammatical elements from reflections (e.g., `[: $refl$ :]`).
```